azure-iot-device/azure/iot/device/iothub/models/message.py

```diff
@@ -52,3 +53,6 @@ def __init__(
         self.content_encoding = content_encoding
         self.content_type = content_type
         self.output_name = output_name
+
+    def __str__(self):
```

Comment:
>__str__ [](start = 8, length = 7)

if there is a string representation why should it not return all the attributes ?

Reply:
It could. That's a discussion we could have. The main point of adding this is just to have SOMETHING now, because in the user study people kept trying to get the data from Message by printing or doing str convert.

---
In reply to: [290987167](https://github.com/Azure/azure-iot-sdk-python-preview/pull/101#discussion_r290987167) [](ancestors = 290987167)